src/domain/metric/service/postMetric.js

```diff
@@ -30,7 +31,7 @@ export default (posts: Post[]): PostMetric[] => {
     return {
       post,
       id: post.id,
-      createdTime: new Date(post.created_time),
+      createdTime: moment(post.created_time).format(),
```

Comment:
All of the reason is because `PostMetric` type is like this:
```
export type PostMetric = {
  post: Post,
  id: string,
  createdTime: Date,
  text: string,
  from?: {
    id: string,
    name: string,
  },
  picture?: string,
  url: string,
  sharesCount: number,
  likesCount: number,
  commentsCount: number,
}
```
Unless you will fix all of files that consumes `PostMetric`.